cmd/history.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
```

Comment:
Nit: I believe we now include the full APACHE-2 license text as the copyright header. (Just look at some recently edited files in the repo. At the very least, this should say 2018.